lib/data_update_scripts/20200917125741_re_index_listing_with_originally_published_at.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class ReIndexListingWithOriginallyPublishedAt
+    def run
+      Listing.find_each(&:index_to_elasticsearch_inline)
```

Comment:
when I run this I'm having the following error

```bash
$ rails data_updates:run

2020-09-17 19:37:10 +0530: PUT http://localhost:9200/classified_listings_development_alias/_doc/1 [status:400, request:0.005s, query:N/A]
2020-09-17 19:37:10 +0530: > {"id":1,"body_markdown":"## Explicabo\nTempora reiciendis dolorem. Est facilis in. Aut consequuntur impedit.\n## Sint\nSint maxime cum. Hic accusantium velit. Qui qui et.\n0. Fuga. \n1. Nemo. \n","bumped_at":"2020-09-17T12:22:38.734Z","category":"cfp","contact_via_connect":true,"expires_at":null,"originally_published_at":"2020-09-17T12:22:38.762Z","location":"Schimmelside","processed_html":"\n  <a href=\"#explicabo\" class=\"anchor\">\n  </a>\n  Explicabo\n\n\n<p>Tempora reiciendis dolorem. Est facilis in. Aut consequuntur impedit.</p>\n\n  <a href=\"#sint\" class=\"anchor\">\n  </a>\n  Sint\n\n\n<p>Sint maxime cum. Hic accusantium velit. Qui qui et.</p>\n\n<ol>\n<li>Fuga. </li>\n<li>Nemo. </li>\n</ol>\n","published":true,"slug":"ipsum-animi-aut-facere-5796","title":"Ipsum animi aut facere.","user_id":1,"tags":["python","discuss"],"author":{"username":"org8016","name":"Bream-Hall","profile_image_90":"/uploads/organization/profile_image/2/0d24410d-3fe9-48a6-ae67-9d1b1deb4464.png"},"last_indexed_at":"2020-09-17T14:07:10.866Z"}
2020-09-17 19:37:10 +0530: < {"error":{"root_cause":[{"type":"strict_dynamic_mapping_exception","reason":"mapping set to strict, dynamic introduction of [originally_published_at] within [_doc] is not allowed"}],"type":"strict_dynamic_mapping_exception","reason":"mapping set to strict, dynamic introduction of [originally_published_at] within [_doc] is not allowed"},"status":400}
2020-09-17 19:37:10 +0530: [400] {"error":{"root_cause":[{"type":"strict_dynamic_mapping_exception","reason":"mapping set to strict, dynamic introduction of [originally_published_at] within [_doc] is not allowed"}],"type":"strict_dynamic_mapping_exception","reason":"mapping set to strict, dynamic introduction of [originally_published_at] within [_doc] is not allowed"},"status":400}

```
any thoughts on how to go about solving this @forem/oss  ? And I feel like I should have done the updating index in batches.

Reply:
I think this error is because of strict indexing.

Reply:
Have you run `bin/setup` recently to update your mappings? This indicates we are trying to index a field that is not in our mapping which is likely because you have not updated them recently